CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Move pre-save UUID manipulations to be before the post-install hooks  
```

Comment:
I'd argue this isn't needed and is encapsulated as part of new feature development of multi-xcodeproj.

Reply:
I wasn't talking about encapsulation though. I have a post-install script that relies on UUIDs being known at the time.  
Specifically, it uses them to mark some targets as non-migratable for Swift so that there are no warnings about targets I can't control.

Reply:
Oh, did you mean the changelog record?

Reply:
yes!

Reply:
Sure, will remove...